frontends/p4/fromv1.0/converters.cpp

```diff
@@ -416,6 +416,35 @@ ExternConverter *ExternConverter::get(cstring type) {
 
 ///////////////////////////////////////////////////////////////
 
+std::map<cstring, std::vector<PrimitiveConverter *>> *PrimitiveConverter::all;
+
+PrimitiveConverter::PrimitiveConverter(cstring name, int prio) : prim_name(name), priority(prio) {
+    static std::map<cstring, std::vector<PrimitiveConverter *>> converters;
```

Comment:
Why do you need another local static variable to initialize 'all'? You could initialize all directly as a static data member to an empty map. And BTW, 'all' is not the most fortunate choice for this name.